test/Base_Tests/src/Semantic/Arrow_Interop_Spec.enso

```diff
@@ -0,0 +1,35 @@
+from Standard.Base import all
+import Standard.Base.Errors.Common.Index_Out_Of_Bounds
+import Standard.Base.Errors.Common.Unsupported_Argument_Types
+from Standard.Test import all
+
+foreign arrow new_arrow = """
+    new[Int8]
+
+Vector.set self index elem =
```

Comment:
There `Polyglot.read_array_element` method, maybe you want to add `write_array_element` there.